server/update_server.sh

```diff
@@ -16,4 +16,21 @@ for f in server/redis-migrations/*.js; do
   node -r "./babel-register" -r "core-js/proposals/reflect-metadata" -r "dotenv/config" "$f" || exit 1
 done
 
+echo "Syncing public assets to the cloud"
+ACCESS_KEY=$(echo $SB_FILE_STORE | jq '.doSpaces.accessKeyId')
+SECRET_KEY=$(echo $SB_FILE_STORE | jq '.doSpaces.secretAccessKey')
+HOST_BASE=$(echo $SB_FILE_STORE | jq '.doSpaces.endpoint')
+HOST_BUCKET="%(bucket)s.${HOST_BASE}"
+SPACE_NAME=$(echo $SB_FILE_STORE | jq '.doSpaces.bucket')
+
+s3cmd sync \
+  --access_key=${ACCESS_KEY} \
+  --secret_key=${SECRET_KEY} \
+  --host=${HOST_BASE} \
+  --host-bucket=${HOST_BUCKET} \
+  --recursive \
+  --delete-removed \
+  --skip-existing \
```

Comment:
I don't know what this argument does, but it sure sounds like "don't bother doing anything with stuff that already exists, even if the content is different", which seems like not something we want. Is that the case?

Reply:
Yeah, I might have misread the description of this argument. I thought it would only skip files whose md5 checksum and size is the same since `sync` command checks those by default. But in the sync guide it says:

_Sometimes you don’t want to compare checksums and sizes of the remote vs local files and only want to upload those that are new. For that use the `—skip-existing` option._

So yeah, we probably don't want this either.